packages/libcpufeatures/build.sh

```diff
@@ -4,6 +4,6 @@ TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="0.8.0"
```

Comment:
Please also bump the revision of this package. `TERMUX_PKG_REVISION=1`

Reply:
Happy to, but does that make sense if I'm right that the only thing that's happened is that upstream force-rewrote a tag? (I'm 75% sure at this point.) Do you want `TERMUX_PKG_REVISION` to still be set after the bot comes around for a future `0.9.0` (or do you expect the bot to remove it)?

Reply:
Actually bumping revision is very necessary. Otherwise CI will fail on uploading packages.

When upgrading packages, bot will automatically remove `TERMUX_PKG_REVISION`.

Reply:
Got it. Done.